src/OpenTelemetry.Instrumentation.EntityFrameworkCore/TracerProviderBuilderExtensions.cs

```diff
@@ -43,13 +45,36 @@ public static class TracerProviderBuilderExtensions
     public static TracerProviderBuilder AddEntityFrameworkCoreInstrumentation(
         this TracerProviderBuilder builder,
         Action<EntityFrameworkInstrumentationOptions> configure)
+    {
+        return AddEntityFrameworkCoreInstrumentation(builder, name: null, configure);
+    }
```

Comment:
nit: It will be good to have common way to call method with all attributes.
Use here  arrow style, or fix `AddEntityFrameworkCoreInstrumentation(this TracerProviderBuilder builder)`

Reply:
Not sure I understand what you mean. I just went with the implementation form [OpenTelemetry.Instrumentation.AspNetCore](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Instrumentation.AspNetCore/TracerProviderBuilderExtensions.cs)

Reply:
![image](https://user-images.githubusercontent.com/5972917/220380840-8f43b445-3441-4eee-b0e4-e167528569ee.png)
1. arrow style
2. method with body.

Doing the same, looks differently. Please unify if possible :)

Reply:
Oh, got it! Will do 😀